chart/kubeapps/templates/_helpers.tpl

```diff
@@ -131,3 +131,10 @@ Create name for the tiller-proxy based on the fullname
 {{- define "kubeapps.tiller-proxy.fullname" -}}
 {{ template "kubeapps.fullname" . }}-internal-tiller-proxy
 {{- end -}}
+
+{{/*
+Create name for the tiller-proxy based on the fullname
+*/}}
+{{- define "kubeapps.custom-helm-registry.secret" -}}
```

Comment:
I thought after reading this https://github.com/kubeapps/kubeapps/issues/515#issuecomment-415203354 that we were going to offer an option per-repository instead of a global option.

Reply:
This is more simple for the moment. Note that having a single secret doesn't mean that we can only add one CA. We can add more files to that secret if we want to implement the UX that @prydonius mention in that comment. I don't know if that's worth it right now though. I think that users right now only have one personal registry.